src/main/scala/org/broadinstitute/dsde/workbench/sam/db/tables/GroupTable.scala

```diff
@@ -21,8 +21,8 @@ object GroupTable extends SQLSyntaxSupport[GroupRecord] {
     rs.get(e.id),
     rs.get(e.name),
     rs.get(e.email),
-    rs.get(e.updatedDate),
-    rs.get(e.synchronizedDate)
+    rs.timestampOpt(e.updatedDate).map(_.toInstant),
```

Comment:
ended up leaving these as `Instant`s instead of switching them to `Date`s because despite what I thought scalike said about[ natively supporting `java.util.Date`](http://scalikejdbc.org/documentation/sql-interpolation.html) (see the 'Natively-handled types' section) the code seems to say otherwise as the [implicit we would use has been commented out](https://github.com/scalikejdbc/scalikejdbc/blob/45fa0ee59e13dce2d93a4dca0aff8a6f529dea84/scalikejdbc-core/src/main/scala/scalikejdbc/TypeBinder.scala#L96) for the past 6 years as it was apparently causing errors. 
`Instant` will work fine for our purposes since it supports millisecond precision.